### ISSUE TITLE
Bug: `civo k8s create cluster_name` no longer works

### ISSUE THREAD

**Issue by kaihoffman** [civo/cli#200]
I used to be able to create a default cluster (3 nodes, etc) by running `civo k8s create clustername`

Now it gives me:

```console
civo k8s create knative-test
Error: You can't use --firewall-rules without --create-firewall flag
```

It works if I provide the `--create-firewall` switch but this is a breaking change from a number of tutorials etc.

**Comment by andyjeffries** [comment]
In a similar vein, the quick create for an instance doesn't work either:

```
$ civo instance create dokku-demo
Error: required flag(s) "diskimage", "size" not set
```

It should default to creating a medium latest Ubuntu instance. If I specify those fields I still get some weirdness...

```
$ civo instance create dokku-demo -t ubuntu-focal -i g3.medium
The instance tiny-oak has been created
```

What!? `tiny-oak`?? What happened to my choice of `dokku-demo`? Apparently this is documented, but unexpected when the K3s command Kai showed just uses it as the first positional parameter and `civo instance show tiny-oak` works as expected. I'd suggest allowing either value - first un-switched parameter if present, if not, then the `-n` flag.

**Comment by alejandrojnm** [comment]
@haardikdharma10 this is done ??

**Comment by haardikdharma10** [comment]
> @haardikdharma10 this is done ??

yes, only cutting a tag is left 😃

**Comment by haardikdharma10** [comment]
Fixed in #203. Closing.
